includes/class-sensei-wp-kses.php

```diff
@@ -92,4 +92,48 @@ public static function maybe_sanitize( $content, $allowed_html )
 
 		return $html_security ? self::wp_kses( $content, $allowed_html ) : $content;
 	}
+
+	/**
+	 * Sanitizes data for allowed HTML tags for an array of elements.
+	 *
+	 * @since 1.12.2
+	 *
+	 * @param array $unescaped_data Array of unescaped data.
+	 * @return array Escaped data.
+	 **/
+	public static function wp_kses_post_array( $unescaped_data ) {
+		$escaped_data = array();
+
+		foreach ( $unescaped_data as $key => $data ) {
+			$escaped_data[$key] = wp_kses_post( $data );
+		}
+
+		return $escaped_data;
+	}
+
+	/**
+	 * Sanitizes content for an array of HTML elements.
+	 * Allows for both regular post HTML tags and custom HTML tags.
+	 *
+	 * @since 1.12.2
+	 *
+	 * @param array $unescaped_data Array of unescaped data.
+	 * @param array $allowed_html List of allowed HTML elements.
```

Comment:
We may want to specify that this will be merged with `wp_kses_allowed_html( 'post' )`.

Reply:
Fixed in https://github.com/Automattic/sensei/pull/2296/commits/3f48b3c575af304ee8c0b29b1ab5e245b7f93463.